include/iotc_error.h

```diff
@@ -21,118 +21,105 @@
+  /** The SDK is not initialized. Run iotc_initialize() and try again. Numeric code: 41 */ IOTC_NOT_INITIALIZED,
+  /** @cond Numeric code: 42 */ IOTC_FAILED_INITIALIZATION, /** @endcond */
+  /** @cond Numeric code: 43 */ IOTC_ALREADY_INITIALIZED, /** @endcond */
+  /** A parameter is missing or invalid. Numeric code: 44 */ IOTC_INVALID_PARAMETER,
+  /** @cond Numeric code: 45 */ IOTC_UNSET_HANDLER_ERROR, /** @endcond */
+  /** Can't create a JSON Web Token because the cryptography library in the Board Support Package (BSP) isn't ported to your platform. To port the cryptography library, customize the <a href="../../bsp/html/d6/d01/iotc__bsp__crypto_8h.html">hardware-specific drivers and routines</a> for your device and then <a href="../../../porting_guide#bsp-code-porting-process">link the library to the client application</a>. <br> Numeric code: 46 */ IOTC_NOT_IMPLEMENTED,
+  /** @cond Numeric code: 47 */ IOTC_ELEMENT_NOT_FOUND, /** @endcond */
+  /** @cond Numeric code: 48 */ IOTC_SERIALIZATION_ERROR, /** @endcond */
+  /** @cond Numeric code: 49 */ IOTC_TRUNCATION_WARNING, /** @endcond */
+  /** @cond Numeric code: 50 */ IOTC_BUFFER_OVERFLOW, /** @endcond */
+  /** @cond Numeric code: 51 */ IOTC_THREAD_ERROR, /** @endcond */
+  /** @cond Numeric code: 52 */ IOTC_NULL_CONTEXT, /** @endcond */
+  /** @cond Numeric code: 53 */ IOTC_NULL_WILL_TOPIC, /** @endcond */
+  /** @cond Numeric code: 54 */ IOTC_NULL_WILL_MESSAGE, /** @endcond */
+  /** @cond Numeric code: 55 */ IOTC_NO_MORE_RESOURCE_AVAILABLE, /** @endcond */
+  /** @cond Numeric code: 56 */ IOTC_FS_RESOURCE_NOT_AVAILABLE, /** @endcond */
+  /** @cond Numeric code: 57 */ IOTC_FS_ERROR, /** @endcond */
+  /** The memory limiter module isn't installed. To enable the memory limiter, set the <a href="../../../porting_guide.md#development-flags"><code>memory_limiter</code></a> flag in a <a href="../../../porting_guide.md#config">CONFIG</a> argument. <br> Numeric code: 58 */ IOTC_NOT_SUPPORTED,
+  /** The event processor stopped because the client application {@link iotc_events_stop() shut down the event engine}. Numeric code: 59 */ IOTC_EVENT_PROCESS_STOPPED,
+  /** @cond Numeric code: 60 */ IOTC_STATE_RESEND, /** @endcond */
+  /** @cond Numeric code: 61 */ IOTC_NULL_HOST, /** @endcond */
+  /** @cond Numeric code: 62 */ IOTC_TLS_FAILED_CERT_ERROR, /** @endcond */
+  /** @cond Numeric code: 63 */ IOTC_FS_OPEN_ERROR, /** @endcond */
+  /** @cond Numeric code: 64 */ IOTC_FS_OPEN_READ_ONLY, /** @endcond */
+  /** @cond Numeric code: 65 */ IOTC_FS_READ_ERROR, /** @endcond */
+  /** @cond Numeric code: 66 */ IOTC_FS_WRITE_ERROR, /** @endcond */
+  /** @cond Numeric code: 67 */ IOTC_FS_CLOSE_ERROR, /** @endcond */
+  /** @cond Numeric code: 68 */ IOTC_FS_REMOVE_ERROR, /** @endcond */
+  /** @cond Numeric code: 69 */ IOTC_NULL_PROJECT_ID_ERROR, /** @endcond */
+  /** The private key isn't signed with ES256. Numeric code: 70 */ IOTC_ALG_NOT_SUPPORTED_ERROR,
```

Comment:
This isn't accurate enough. I'd write something like this: The private key's crypto algorithm type provided for JWT signature isn't supported.

Reply:
Done